src/pages/Contributors/OrganizationSearch.js

```diff
@@ -80,13 +80,17 @@ const OrganizationSearch = ({
             fullWidth
             handleHomeEndKeys
             inputValue={inputValue}
+            ListboxProps={{
+              'data-cy': 'organization-search-list'
+            }}
```

Comment:
Eslint missing trailing comma warning here.  Could this prop be compacted to a single line?